modules/multy-s3-bucket/0.2.0/s3-primary-bucket.tf

```diff
@@ -33,13 +33,102 @@ resource "aws_s3_bucket_lifecycle_configuration" "primary" {
   provider   = aws.primaryregion
   depends_on = [aws_s3_bucket_versioning.primary]
   bucket     = aws_s3_bucket.primary.id
-  rule {
-    id = "expire_non_current_version"
 
-    filter {}
+  dynamic "rule" {
+
+    for_each = aws_route53_zone.clusters
+    content {
+      id = "expire_old_vault_backups"
+
+      filter {
+        prefix = "${rule.key}/hashicorp-vault-backups/"
+      }
+
+      expiration {
+        days = 15
+      }
+
+      status = "Enabled"
+    }
+    
+  }
+
+  dynamic "rule" {
+
+    for_each = aws_route53_zone.clusters
+    content {
+      id = "expire_old_tls_backups"
+
+      filter {
+        prefix = "${rule.key}/tls-cert-backups/"
+      }
+
+      expiration {
+        days = 15
+      }
+
+      status = "Enabled"
+    }
+   
+  }
+
+  dynamic "rule" {
+    id = "expire_transition_vault"
+    for_each = aws_route53_zone.clusters
+
+    content {
```

Comment:
You have content **here** but not here: https://github.com/GlueOps/terraform-module-cloud-multy-prerequisites/pull/376/files#diff-1db77e282d9bbe89e70258b80560c7dd7700820313773f59e73062857c4ceb27R112

Reply:
Is content {} needed?

Reply:
👍🏾